### ISSUE TITLE
Make text on about page wider?

### ISSUE THREAD

**Issue by SensusOlfactus** [CollectionBuilder/dhsi-demo#10]
Is there a way to make the text on the about page wider?

**Comment by dcnb** [comment]
YES!  in your _sass folder, there is a file called pages.scss with an about section at the bottom. If you look at it (I'll paste it way below), you can see there is a "max-width" value set for headers, paragraphs, and some specific classes. To override those settings, either edit that file or just change the below to edit the 650px option to a larger width and then add it to your _custom.scss file (the latter is probably recommended).  

    h1, h2, h3, h4, h5, h6 {
        max-width: 650px;
    }
    
    p, .about-narrowed-content, .footnotes, ol, ul {
        max-width: 650px;
    }


---Below is the content of that About section in the _pages.scss file ---

/* About */
/* creating a narrow text column */ 
.about-contents > {
    h2 {
        font-weight: bold;
        padding-top: 100px;
        border-top: 1px dotted black;
    }
    
    h1, h2, h3, h4, h5, h6 {
        max-width: 650px;
        margin: 1.5rem auto 1rem;
    }
    
    p, .about-narrowed-content, .footnotes, ol, ul {
        margin-left: auto;
        margin-right: auto;
        max-width: 650px;
    }

}

**Comment by SensusOlfactus** [comment]
Thank you! I'll try it.

Dr. Katelynn Robinson

***@***.***

@SensusOlfactus on Twitter

https://www.routledge.com/The-Sense-of-Smell-in-the-Middle-Ages-A-Source-of-Certainty/Robinson/p/book/9780367000684


________________________________

**Comment by evanwill** [comment]
Or if you just want your About to be a full contain width page (like the home page), change `layout: about` to `layout: page` in the front matter of your about page!

**Comment by dcnb** [comment]
Slight edit to css above. I missed a top section that nests the code in our about-contents class. If you want to edit the _custom.scss file, edit the below instead of what was included earlier. Sorry!

.about-contents > {
h1, h2, h3, h4, h5, h6 {
    max-width: 650px;
}

p, .about-narrowed-content, .footnotes, ol, ul {
    max-width: 650px;
}
}
